.github/workflows/release.yml

```diff
@@ -14,9 +14,11 @@ jobs:
     runs-on: windows-2022
     strategy:
       matrix:
-        python: ["3.9", "3.10"]
+        python: ["3.7", "3.9", "3.10"]
         platform: [x64, win32]
         vstudio: [17]
+        exclude:
+          - python: ${{ github.event.inputs.ref == 'latest' && '3.7' }}
```

Comment:
What does this do?

Reply:
It's a bit clunky but essentially it allows having two gvsbuild stacks, one with slightly older versions of everything that uses python 3.7 and one that uses newer versions and builds with python 3.9 and 3.10.

You build both by running the workflow manually using "older" and "latest" as the checkout ref.

I expect eventually "older" would be dropped, but I wanted to include it for now because it's what most closely resembles the current builds.